pp/series_data/tests/querier/instant_querier_tests.cpp

```diff
@@ -56,10 +58,13 @@ TEST_P(InstantQuerierOpenChunkFixture, InstantQueryOpenChunk) {
   encoder_.encode(0, 5, 5.0);
 
   // Act
-  series_data::InstantQuerier::query_sample(sample_, storage_, GetParam().request.ls_id, GetParam().request.timestamp);
+  std::vector<Sample> samples = {sample_};
+  std::vector<uint32_t> ls_ids = {GetParam().request.ls_id};
+  series_data::InstantQuerier instant_querier(storage_);
+  instant_querier.query(samples, ls_ids, GetParam().request.timestamp);
```

Comment:
Unit test should should be as small as possible, but should be easy to read. Act section should contains only actions, not arrange code
```suggestion
  InstantQuerier{storage_}.query(samples, std::vector{{GetParam().request.ls_id}}, GetParam().request.timestamp);
```